File: benchmarks/suite5/nested.c
#include <stdint.h>
#include <stdio.h>
#include "bench.h"

int main(void) {
  const int64_t n = 450;
  int64_t s = 0;
  for (int64_t i = 0; i < n; i++) {
    for (int64_t j = 0; j < n; j++) {
      s = s + (i * j + i) % 97;
    }
  }
  suite5_print_i64(s);
  return 0;
}

File: benchmarks/suite5/nested.zig
const bench = @import("bench.zig");
const clib = @cImport({
    @cInclude("stdio.h");
});

pub fn main() void {
    const n: i64 = 450;
    var s: i64 = 0;
    var i: i64 = 0;
    while (i < n) : (i += 1) {
        var j: i64 = 0;
        while (j < n) : (j += 1) {
            s = s + @rem(i * j + i, 97);
        }
    }
    bench.printI64(s);
}
